buildpipeline/pipelines.json

```diff
@@ -24,6 +24,19 @@
           }
         },
         {
+          "Name": "DotNet-CoreClr-Trusted-Linux",
+          "Parameters": {
+            "DockerTag": "centos-6-783abde-20171304101322",
```

Comment:
We don't have license for redhat 6, and have to use Cento 6 as the replacement as it's free and almost identical to redhat 6.